app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/LearningSettingsActivityTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.github.HumanLearning2021.HumanLearningApp
+
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.HumanLearning2021.HumanLearningApp.view.LearningActivity
+import com.github.HumanLearning2021.HumanLearningApp.view.LearningSettingsActivity
+import com.schibsted.spain.barista.assertion.BaristaClickableAssertions.assertNotClickable
+import com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertDisplayed
+import com.schibsted.spain.barista.interaction.BaristaClickInteractions.clickOn
+import com.schibsted.spain.barista.interaction.BaristaClickInteractions.longClickOn
+import org.junit.*
+import org.junit.runner.RunWith
+import org.junit.runners.MethodSorters
+
+class LearningSettingsActivityTest {
+    @RunWith(AndroidJUnit4::class)
+    @FixMethodOrder(MethodSorters.NAME_ASCENDING) // to enforce consistent order of tests
+    class AddPictureActivityTest {
+
+        @get:Rule
+        val testRule = ActivityScenarioRule(LearningSettingsActivity::class.java)
+
+
+        @Before
+        fun setUp() {
+            Intents.init()
+            // By waiting before the test starts, it allows time for the app to startup to prevent the
+            // following error to appear on cirrus:
+            // `Waited for the root of the view hierarchy to have window focus and not request layout for 10 seconds.`
+            // This solution is not ideal because it slows down the tests, and it might not work
+            // every time. But there isn't a better solution that I (Niels Lachat) know of.
+            val delayBeforeTestStart: Long = 3000
+            TestUtils.waitFor(delayBeforeTestStart)
+        }
+
+
+        @After
+        fun cleanUp() {
+            Intents.release()
+            testRule.scenario.close()
+        }
+
+        @Test
+        fun bothButtonsAreDisplayed() {
+            assertDisplayed(R.id.btChoosePresentation)
+            assertDisplayed(R.id.btChooseRepresentation)
+        }
+
+        @Test
+        fun learningModeTextViewIsDisplayed() {
+            assertDisplayed(R.id.tvMode)
+        }
```

Comment:
I'm not 100% sure but from what I understand, each time a test case starts, it has to restart the activity and that takes some time. So maybe to speedup the tests it would be better to put all your `assertDisplayed` tests in a same test case (since it essentially tests the same thing)